minification.ipynb

```diff
@@ -0,0 +1,993 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    scvi.model.SCVI.load("local/hca", adata=model.adata)
</pre><p>I would try, except this part for the key error and then print the key error. </p><p> </p><p>This could allow us one day to test these tutorials more rigourously</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scvi-tutorials/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>